lib/src/migrator.dart

```diff
@@ -126,10 +127,45 @@ class _Migrator extends RecursiveStatementVisitor implements ExpressionVisitor {
   /// Adds a namespace to any function call that require it.
   void visitFunctionExpression(FunctionExpression node) {
     visitInterpolation(node.name);
+    _patchNamespaceForFunction(node.name.asPlain, (name, namespace) {
+      _currentMigration.patches.add(Patch(node.name.span, "$namespace.$name"));
+    });
     visitArgumentInvocation(node.arguments);
 
-    if (node.name.asPlain == null) return;
-    var name = node.name.asPlain;
+    if (node.name.asPlain == "get-function") {
+      var nameArgument =
+          node.arguments.named['name'] ?? node.arguments.positional.first;
+      if (nameArgument is! StringExpression ||
+          (nameArgument as StringExpression).text.asPlain == null) {
+        print(nameArgument.span.message(
+            "WARNING - get-function call may require \$module parameter"));
```

Comment:
It's probably a good idea to print this to stderr.

Reply:
Is there an equivalent to the `prints` matcher for testing stderr?

Reply:
Unfortunately no there isn't. Usually when I'm testing command-line executables, I just run them via `Process.runSync()` or [the `test_process` package](https://pub.dartlang.org/packages/test_process) so I don't have to worry about mocking out all the APIs.

Reply:
Okay, in that case, I'll leave this as is for now. I've made an issue (#14) for this, which I'll handle in a separate PR